OvmfPkg/PlatformCI/PlatformBuildLib.py

```diff
@@ -207,6 +207,7 @@ def FlashRomImage(self):
         args += " -global isa-debugcon.iobase=0x402"                        # debug messages out thru virtual io port
         args += " -net none"                                                # turn off network
         args += " -smp 4"
+        args += " -cpu qemu64,+rdrand"

```

Comment:
I'd suggest to use `-cpu IvyBridge` instead (first intel cpu with rdrand support according to wikipedia), or something newer.

`qemu64` is indeed the default, for historical and compatibility reasons, but it is a bad choice because it lacks a bunch of modern cpu features.

Reply:
@kraxel Sure, I'm happy to add whatever cpu you would like, I was just attempting to make the minimal change here, but I agree there are better choices. Would you prefer IvyBridge or something newer?

Reply:
Don't care much about the exact model.  So lets go with IvyBridge.  Adding a comment mentioning this was picked for rdrand support is probably a good idea too.